docs/tutorial.rst

```diff
@@ -198,6 +198,8 @@ Hy.  Let's experiment with this in the hy interpreter::
   {'dog': 'bark', 'cat': 'meow'}
   => (, 1 2 3)
   (1, 2, 3)
+  => #{3 1 2}
+  set([1L, 2L, 3L])
```

Comment:
@zackmdavis I had intentionally used the Python 2 version to show the difference between the similar-looking dict literal. Of course, if you want to change it, I'll do it. :)